walkeeper/src/wal_service.rs

```diff
@@ -33,9 +34,19 @@ pub fn thread_main(conf: SafeKeeperConf, listener: TcpListener) -> Result<()> {
     }
 }
 
+// Get unique thread id (Rust internal), with ThreadId removed for shorter printing
+fn get_tid() -> u64 {
```

Comment:
Just a note: tracing formatter can print thread names and ids by default, we can enable either on a global level, if needed: https://github.com/tokio-rs/tracing/blob/5a076ed69e1ca8b403a4f3b34223d846946352a1/tracing-subscriber/src/fmt/fmt_subscriber.rs#L313

Reply:
> tracing formatter can print thread names and ids by default,

Cool, didn't know that. However, it is verbose (`ThreadId(04)` instead of `tid=4`, not sure I want to thread ids that much... the original motivation was to see all logs from one connection (thread), e.g. where it ends. I'm inclined to leave these tid's for now and re-enable tracing support globally if someone wants it.

Reply:
>  is it right that we cannot use it because it's nightly-only?

Yeah. Nice, I actually looked up that seval months ago and didn't find it at all.